cpp/inc/bond/protocol/simple_binary.h

```diff
@@ -36,8 +36,8 @@
                      string, wstring        | count | characters |
                                             '-------'------------'
 
-                           count            variable encoded uint32 count of 1-byte (for
-                                            string) or 2-byte (for wstring) Unicode code
+                           count            uint32 count of 1-byte (for string)
+                                            or 2-byte (for wstring) Unicode code
                                             units (variable encoded in v2)
```

Comment:
The C++ API reference pages are [automatically rebuilt](https://github.com/Microsoft/bond/blob/5391636d5b4a8c59981527822841649a292d6be6/appveyor.yml#L232-L259) when a push to master is made. They're generated from doc comments in [the C++ files using Doxygen](https://github.com/Microsoft/bond/blob/5391636d5b4a8c59981527822841649a292d6be6/doc/CMakeLists.txt#L129-L153).